plugin.cpp

```diff
@@ -102,6 +111,17 @@ void setPluginConfig(Random *random, ConfigCategory *config)
 	}
 	random->setNumAssets(nAssets);
 
+	unsigned int nDatapoints = 1;
+	if (config->itemExists("numDatapoints"))
+	{
+		nDatapoints = stoul(config->getValue("numDatapoints"), nullptr, 0);
+		if (nDatapoints <= 0)
+		{
+			throw runtime_error("The value of numDatapoints, number of unique data points to simulate, must be greater than 0");
```

Comment:
The value of numDatapoints, number of unique data points to simulate, must be greater than 0 -> The number of unique data points to simulate must be greater than 0